core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.scss

```diff
@@ -0,0 +1,30 @@
+@use "variables" as *;
+
+form {
+    margin: $spacing-4;
+
+    .p-input-icon-right {
+        width: 100%;
+        position: relative;
+
+        input {
+            width: 100%;
+        }
+
+        button,
+        .pending-label {
+            position: absolute;
+            right: 0;
+            top: 0;
+            bottom: 0;
+            margin: auto 0;
+            background: none;
+        }
+
+        .pending-label {
+            height: fit-content;
+            font-size: 12px;
+            margin-right: 0.5rem;
```

Comment:
We have variables for this: `core-web/libs/dotcms-scss/shared/_spacing.scss` and `core-web/libs/dotcms-scss/shared/_fonts.scss`

You just need to call the variables in the SCSS file like this:

```
@use "variables" as *;
```

Example: `core-web/libs/block-editor/src/lib/extensions/asset-form/asset-form.component.scss`